cfme/tests/infrastructure/test_quota_tagging.py

```diff
@@ -5,15 +5,27 @@
 from cfme.configure.configuration.region_settings import Tag, Category
 from cfme.infrastructure.provider.rhevm import RHEVMProvider
 from cfme.infrastructure.provider.virtualcenter import VMwareProvider
+from cfme.markers.env_markers.provider import ONE_PER_TYPE
 from cfme.provisioning import do_vm_provisioning
+from cfme.services.service_catalogs import ServiceCatalogs
+from cfme.utils.appliance import ViaSSUI, ViaUI
 from cfme.utils.generators import random_vm_name
 from cfme.utils.update import update
 
+from widgetastic.utils import partial_match
+
 pytestmark = [
-    pytest.mark.provider([RHEVMProvider, VMwareProvider], scope="module")
+    pytest.mark.provider([RHEVMProvider, VMwareProvider], scope="module", selector=ONE_PER_TYPE)
 ]
 
 
+@pytest.fixture
+def admin_email(appliance):
```

Comment:
Question: Why we need this fixture for?

Reply:
When we run user tagging related test case, we need to update the user email id ( if its not present in the user.) otherwise the request we generated goes into pending state and thus it is required for user tagging related test case.

Reply:
**Required**

@anurag03 Please add a docblock to the fixture explaining its need/use.